src/components/Inputbox.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react'
+import "./Inputbox.css"
+
+export const Inputbox = ({allPosts, setAllPosts}) => {
+  const [post, setPost] = useState("");
+  const [count, setCount] = useState(0);
+
+  const submitPost =()=>{
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', {
+    method:'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      message:post,
+    })
+  })
+    .then(res=>res.json())
+    .then(json=>{
+      setAllPosts([json, ...allPosts])
+      setPost("");
+      setCount("0");
+    })
+    .catch(error=>{
+      console.error('Error:', error)
+    }) 
+  }
+
+  const getClass =()=>{
+    if (count< 5 && count >0 ){
+      return 'out-of-range'
+    } else if(count > 140){
+      return 'out-of-range'
+    } else {
+      return 'count'
+    }
+  }
+
+  return (
+    <div className='inputbox'>
+      <h4>What make you happy right now? </h4>
```

Comment:
you are being "punished" for not following a sequentially-descending order on your headings - i guess you could used h2 instead of h4 and omit this problem.